manual/output_overview.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Output overview}
+\label{chap:output_overview}
+
+%% Detail contents of output files.
+QMCPACK writes several output files which report information about the simulation (e.g. the physical properties such as the energy), as well as information about the computational aspects of the simulation, checkpoints, and restarts.
+The types of output files generated depend on the details of a calculation. The list below is not meant to be exhaustive, but rather to highlight some salient features of the more common filetypes. Further detail can be found in the description of the the estimator one is interested in computing.
+
+
+\section{The .scalar.dat file}
+\label{sec:scalardat_file}
+The most important output file is the \texttt{.scalar.dat} file. This file contains the output of block averaged properties of the system such as the local energy and other estimators.
+Each line corresponds to an average over $N_{walkers}*N_{steps}$ samples.
+By default, the quantities reported in the \texttt{.scalar.dat} file include:
+
+\begin{description}
+\item[LocalEnergy] The local energy.
+\item[LocalEnergy\_sq] The local energy squared.
+\item[LocalPotential] The local potential energy.
+\item[Kinetic] The kinetic energy.
+\item[ElecElec] The electron-electron potential energy.
+\item[IonIon] The ion-ion potential energy.
+\item[LocalECP] The energy due to the pseudopotential/effective core potential.
+\item[NonLocalECP] The non-local energy due to the pseudopotential/effective core potential.
+\item[MPC] The modified periodic coulomb potential energy.
+\item[BlockWeight] The number of MC samples in the block.
+\item[BlockCPU] The number of seconds to compute the block.
+\item[AcceptRatio] The acceptance ratio.
+\end{description}
+
+\section{The .opt.xml file}
+\label{sec:optxml_file}
+This file is generated after a VMC wave function optimization, and stores the optimized jastrow.
+Conveniently, this file is already formatted in .xml, so it can easily be incorporated into a DMC input file.
```

Comment:
Perhaps should say instead that this file contains the <wavefunction> part of the input file with optimized parameters. This would provide a little more guidance to the user on how it might be used (vs. xml formatted)